### ISSUE TITLE
Link not working

### ISSUE THREAD

**Issue by jdanielricher** [cryptoanarchywiki/cryptoanarchywiki.github.io#11]
In the Cypherpunks Mailing List section, The Cyphernomicon (Cypherpunks Mailing List FAQ), the linked site (www.cypherpunks.to) refuses to connect.

**Comment by tombusby** [comment]
Yeah it seems it's gone down, I've been waiting to see if it goes back up,
but looks like i'll have to switch the link to internet archive.
